it/controllers/TooManyResultsISpec.scala

```diff
@@ -0,0 +1,203 @@
+            doc.h1.text shouldBe tooManyResultsMessages.heading2
+            doc.select("t").text shouldBe tooManyResultsMessages.line1
+            doc.bulletPointList.select("li").first.text shouldBe tooManyResultsMessages.bullet1(testPostCode)
+            doc.bulletPointList.select("li").last.text shouldBe tooManyResultsMessages.bullet2WithFilter(testFilterValue)
+            doc.submitButton.text shouldBe tooManyResultsMessages.button
+            doc.link("enterManual").text shouldBe JourneyConfigDefaults.EDIT_LINK_TEXT
+          }
+        }
+      }
+
+      "the back buttons are not enabled in the journey config" when {
+        "no filter has been entered" when {
+          "the backend service returns too many addresses " in {
+            stubKeystore(testJourneyId, testSelectConfigNoBackButtons, OK)
+            stubGetAddressFromBE(addressJson = addressRecordJsonList(numberOfRepeats = 51))
+
+            val res = buildClientLookupAddress(path = "select?postcode=AB11+1AB&filter=")
+              .withHeaders(HeaderNames.COOKIE -> sessionCookieWithCSRF, "Csrf-Token" -> "nocheck")
+              .get()
+
+            await(res).status shouldBe OK
+
+            val doc = getDocFromResponse(res)
+
+            doc.select("a[class=back-link]") shouldNot have(
+              text("Back")
+            )
+            doc.title shouldBe tooManyResultsMessages.title
+            doc.h1.text shouldBe tooManyResultsMessages.heading1
+            doc.select("t").text shouldBe tooManyResultsMessages.line1
+            doc.bulletPointList.select("li").first.text shouldBe tooManyResultsMessages.bullet1(testPostCode)
+            doc.bulletPointList.select("li").last.text shouldBe tooManyResultsMessages.bullet2NoFilter
+            doc.submitButton.text shouldBe tooManyResultsMessages.button
+            doc.link("enterManual").text shouldBe JourneyConfigDefaults.EDIT_LINK_TEXT
+          }
+        }
+
+        "a filter has been entered" when {
+          "the backend returns too many addresses" in {
+            stubKeystore(testJourneyId, testSelectConfigNoBackButtons, OK)
+            stubKeystoreSave(testJourneyId, testConfigWithoutAddressAsJson, OK)
+            stubGetAddressFromBEWithFilter(addressJson = Json.arr())
+            stubGetAddressFromBE(addressJson = addressRecordJsonList(51))
+
+            val res = buildClientLookupAddress(path = s"select?postcode=AB11+1AB&filter=$testFilterValue")
+              .withHeaders(HeaderNames.COOKIE -> sessionCookieWithCSRF, "Csrf-Token" -> "nocheck")
+              .get()
+
+            await(res).status shouldBe OK
+
+            val doc = getDocFromResponse(res)
+
+            doc.select("a[class=back-link]") shouldNot have(
+              text("Back")
+            )
+            doc.title shouldBe tooManyResultsMessages.title
+            doc.h1.text shouldBe tooManyResultsMessages.heading2
+            doc.select("t").text shouldBe tooManyResultsMessages.line1
```

Comment:
Could you change this to a "p" tag in the view & here. T is not a valid HTML tag & style wise it will default to rendering a P tag if it doesn't understand it, so it's a safe change.